Competition2017/src/competition/subsystems/fuel_launcher/FuelLauncherSubsystem.java

```diff
@@ -0,0 +1,185 @@
+        
+        //encoderCodesProperty = propManager.createPersistentProperty("Launcher Encoder TPC", 4096);
+
+        p = propManager.createPersistentProperty("FuelLauncher wheel P", 0);
+        i = propManager.createPersistentProperty("FuelLauncher wheel I", 0);
+        d = propManager.createPersistentProperty("FuelLauncher wheel D", 0);
+        f = propManager.createPersistentProperty("FuelLauncher wheel F", 0);
+        
+        // Reading through the manual, it looks like only a few things support human units. It might be easier to just keep everything in native units per 100ms,
+        // which is what the Talon uses for all its calculations.
+        // I call this Ticks per Centisecond, or TPC
+        shooterSpeedThresh = propManager.createPersistentProperty("FuelLauncher nominal speed thresh (TPC)", 1);
+        shooterCurrentSpeed = propManager.createEphemeralProperty("FuelLauncher Current Speed (TPC)", 0);
+        shooterTargetSpeed = propManager.createEphemeralProperty("FuelLauncher Goal Speed (TPC)", 0);
+        shooterOutputPower = propManager.createEphemeralProperty("FuelLauncher Voltage", 0);
+        atSpeedProp = propManager.createEphemeralProperty("Is FuelLauncher at speed?", false);
+        shooterTalonError = propManager.createEphemeralProperty("FuelLauncher Speed error", 0);
+        
+        this.masterMotor = factory.getCANTalonSpeedController(1);
+        //this.slaveMotor = factory.getCANTalonSpeedController(8);
+        initiallyConfigureMotorTeam(masterMotor);
+        masterMotor.createTelemetryProperties("FuelLauncher master", propManager);
+        //slaveMotor.createTelemetryProperties("FuelLauncher slave", propManager);
+    }
+
+    private void initiallyConfigureMotorTeam(XCANTalon master) {
+        // TODO: Check faults and voltage/temp/current
+        log.info("Configuring motor");
+        // Master config
+        master.setFeedbackDevice(FeedbackDevice.QuadEncoder);
+        //master.setBrakeEnableDuringNeutral(false);
+        // TODO: move inversion
+                
+        master.reverseSensor(true);
+        master.setInverted(true);
+        //master.enableLimitSwitches(false, false);
+        log.info("Configuring voltages");
+        master.configNominalOutputVoltage(0,  -0);
+        master.configPeakOutputVoltage(12, -12);
+
+        master.setProfile(0);
+        master.setF(.099);
+        updateMotorConfig(master);
+        master.setControlMode(TalonControlMode.Speed);
+        //master.setControlMode(TalonControlMode.PercentVbus);
+
+        master.set(0);
+        
+        // Slave config
+        /*slave.configNominalOutputVoltage(0,  -0);
+        slave.configPeakOutputVoltage(12, -12);
+        slave.enableLimitSwitches(false, false);
+        
+        slave.setControlMode(TalonControlMode.Follower);
+        slave.set(master.getDeviceID());*/
+    }
+    
+    private void updateMotorConfig(XCANTalon motor) {
+        motor.setP(p.get());
```

Comment:
Instead of calling this "configuration", can we call it "control params" or similar to differentiate from one-time set-up?